_maps/map_files/Delta/delta.dmm

```diff
@@ -78721,12 +78721,10 @@
 /area/station/public/fitness)
 "hQz" = (
 /obj/machinery/door/window/classic/normal{
-	name = "Robotics Desk";
-	dir = 8
-	},
-/obj/effect/mapping_helpers/airlock/windoor/access/any/science/robotics{
```

Comment:
It looks like the access helper for the indoors was removed.

Reply:
Just fixed it 
![Screenshot 2024-01-06 164054](https://github.com/ParadiseSS13/Paradise/assets/98861947/54e9f77f-2a9a-48b6-9549-0e1ef994edbc)

Reply:
with robotics airlock helper